documentation/user_docs/docs/manual/Symmetrising_etc.rst

```diff
@@ -1,112 +1,267 @@
+the result.
+
+Symmetric cuts
+--------------
+
+In order to do a symmetrised cut, the ordinary ``cut`` function (see :ref:cut) is used with the appropriate symmetry
+operations additionally passed into the function as an argument after the bin axes specification (see example
+below). The ``cut`` operation will then use the symmetry operations to compute the transformations of the given
+projection, accumulate the symmetrically-related pixels into the primary binning axes and transform their pixel
+coordinates according to the symmetry operations as though the S(**Q**, w) had been wholly symmetrised.
+
+::
+
+   >> w1 = sqw(data);
+   >> sym = SymopReflection([0 1 0], [0 0 1]);
+
+   % Cut from 1 to 3 in X, accumulated with -1 to -3 reflected across the X-axis
+   >> w2 = cut(w1, ortho_proj([1 0 0], [0 1 0]), [1 0.1 3], [-inf inf], [-inf inf], [-inf inf], sym);
+   >> w3 = cut(w1, ortho_proj([1 0 0], [0 1 0]), [1 0.1 3], [1 0.1 3], [-inf inf], [-inf inf], sym)
 
-Symmetrising
-************
+Symmetrise entire SQW
+=====================
 
+``symmetrise_sqw``
+------------------
 
-Symmetrising data in memory
----------------------------
+It is possible to reduce an entire dataset at once by symmetry, transforming all pixels according to the symmetry
+operations and accumulating the transformed pixels into the bins appropriately. This is done through the
+``symmetrise_sqw`` function, the signature for which is below:
 
 ::
 
-   v1=[0,1,0]; v2=[0,0,1]; v3=[0,0,0];
-   wout=symmetrise_sqw(win,v1,v2,v3);
+   >> w1 = sqw(data);
+   >> sym = SymopReflection([0 1 0], [0 0 1]); % Reflect about X-axis
+   >> w2 = symmetrise_sqw(w1, sym);
+
+It is also possible to reduce data through a rotationally symmetric operation, this can be done in one of 2 ways:
+
+::
+
+   >> w1 = sqw(data);
+   >> sym = SymopRotation([0 0 1], 60);  % Perform a 6-fold symmetric roation about Z
+   >> w2 = symmetrise(w1, sym);
+   >> sym = SymopRotation.fold(6, [0 0 1]); % Same as above
+   >> w3 = symmetrise(w1, sym);
+   >> equal_to_tol(w2, w3);
+
+     ans =
 
+       logical
 
-The object ``wout`` is reflected in the plane specified by the vectors ``v1``, ``v2``, and ``v3``. ``v1`` and ``v2`` are vectors which lie parallel to the plane but are not parallel to one another. ``v3`` specifies the offset from Q=(0,0,0) of the reflection plane. In the above example the plane reflects (-1,0,0) to (1,0,0).
+        1
 
-As you can notice, ``symmetrise_sqw`` works on *sqw* objects, stored in memory (the Matlab workspace). Therefore the size in memory of a given object can often act as a limitation on what data can be symmetrised. e.g. a large 3d volume may contain information from so many detector pixels that symmetrising it requires more memory than is available on your computer.
+A ``SymopRotation`` attempts to map the data into the positive quadrant (though for angle > 90 degrees | folds < 4, this
+will fill the positive quadrant and spill into a negative domain).
```

Comment:
this may answer some of my questions above, and if so should be put earlier when describing symoprotation.

This spilling into a negative domain sounds pathological; avoid or fix?